src/backend/InvenTree/common/models.py

```diff
@@ -1930,6 +1930,32 @@ def save(self, *args, **kwargs):
             'default': False,
             'validator': bool,
         },
+        'LOGIN_ENABLE_SSO_GROUP_SYNC': {
+            'name': _('Enable SSO group sync'),
+            'description': _(
+                'Enable synchronizing InvenTree groups with groups provided by the IdP'
+            ),
+            'default': False,
+            'validator': bool,
+        },
+        'SSO_GROUP_KEY': {
+            'name': 'SSO group key',
+            'description': 'The name of the groups claim attribute provided by the IdP',
```

Comment:
Please make these strings translateable, by using the `_('The name of the groups...')` approach

Reply:
The "name" and "description" field for each should be wrapped like that.

Reply:
resolved in d6b6e241584c72905865e2176ea7212b9fbe89d1